docs/blocks/buy-template.md

```diff
@@ -0,0 +1,25 @@
+## Buy
+
+Definition of the output of the buy block. Authoring has to done via templates/buy.docx.
```

Comment:
"has to done via" => "has to be done via"

Might be worthwhile to mention explicitly that these are files in Sharepoint (in the other cases as well).